### ISSUE TITLE
`riak-cs version` doesn't work

### ISSUE THREAD

**Issue by reiddraper** [basho/riak_cs#635]

**Comment by hectcastro** [comment]
May be related: https://github.com/basho/node_package/pull/76

**Comment by reiddraper** [comment]
We're using node_package 1.3.1 now, I'll make a PR to give 1.3.2 a try.

**Comment by reiddraper** [comment]
PR for node_package 1.3.2 in #638.

**Comment by reiddraper** [comment]
This doesn't seem to fix this `riak-cs version` issue for me. Can someone else confirm?

**Comment by kuenishi** [comment]
So do I at release/1.4 HEAD b115afd559330. Using Debian Linux.

```
$ bin/riak-cs version

$ 
```

**Comment by kuenishi** [comment]
After the package build, I succeeded.

```
$ sudo which riak-cs
/usr/sbin/riak-cs
$ sudo riak-cs version
1.3.1.221.gb115afd
```

Still 1.3.1 :P

**Comment by reiddraper** [comment]
@jaredmorrow mentioned to me that this will only work in the packages, not devrel builds.

**Comment by reiddraper** [comment]
Oh, and it's still 1.3.1 because there is no more recent tag. That should be fixed once there is a 1.4rc tag or a proper 1.4.0 tag. I think it just uses git describe or something.

**Comment by jaredmorrow** [comment]
```
➜  /tmp  s3cmd get s3://<daturl>
s3:/<dat-url> -> ./riak-cs-1.4.0rc1-OSX-x86_64.tar.gz  [1 of 1]
 22945947 of 22945947   100% in   10s     2.03 MB/s  done
➜  /tmp  tar -zxf riak-cs-1.4.0rc1-OSX-x86_64.tar.gz
➜  /tmp  cd riak-cs-1.4.0rc1 
➜  riak-cs-1.4.0rc1  ./bin/riak-cs version 
1.4.0rc1
```
